assets/html/style.css

```diff
@@ -0,0 +1,323 @@
+/* fonts */
+body, input {
+  /*font-family: 'Ubuntu', Helvetica, Arial, sans-serif;*/
+  font-family: Helvetica, Arial, sans-serif;
+}
+
+pre, code {
+  /*font-family: 'Ubuntu Mono', 'Courier New', 'Courier', monospace;*/
+  font-family: 'Courier New', 'Courier', monospace;
+}
+
+.fa {
+    font-family: 'FontAwesome';
+}
+
+body {
+    line-height: 1.5;
+}
+
+/*p, pre, ul, ol {
+    margin: 0.5em 0;
+}*/
+
+h1 { font-size: 1.75em; }
+h2 { font-size: 1.50em; }
+h3 { font-size: 1.25em; }
+h4 { font-size: 1.15em; }
+h5 { font-size: 1.10em; }
+h6 { font-size: 1em; }
+
+h4, h5, h6 {
+    margin: 1em 0;
+}
+
+table {
+    border-collapse: collapse;
+    margin: 1em 0;
+}
+
+th, td {
+    border: 1px solid #e1e4e5;
+    padding: 0.5em 1em;
+}
+
+th {
+    border-bottom-width: 2px;
+}
+
+tr:nth-child(even) {
+    background-color: #f3f6f6;
+}
+
+a {
+    text-decoration: none;
+}
+
+/* Inline code and code blocks */
+
+pre, code {
+    /*background-color: #f7f7f7;*/
+    /*color: #73261e;*/
+    background-color: rgba(0,0,0,.04);
+    border-radius: 3px;
+}
+
+pre {
+    padding: 0.5em;
+    overflow: auto;
+}
+
+code {
+    padding: 0.1em;
+}
+
+pre code {
+    background-color: initial;
+    padding: 0;
+}
```

Comment:
Code blocks that have highlight.js highlighting enabled seem to have two background colours, one in front of the other, with a small amount of padding between them.

![2016-08-10-212035_4966x1080_scrot](https://cloud.githubusercontent.com/assets/6144086/17567663/927f9bc0-5f40-11e6-9241-1d360754082e.png)

Note the lighter grey border around the code block.